### PR TITLE
remove WellKnownTopologyKey from PV Topology

### PR DESCRIPTION
**Pull request by Elbehery**
**Is this a bug fix or adding new feature?**
bug fix
**What is this PR about? / Why do we need it?**
fixes https://github.com/kubernetes/kubernetes/issues/102248
**What testing is done?** 
modified testing and verified

### CODE REVIEW

**Comment by Elbehery** [comment]
@jsafrane

**Comment by jsafrane** [comment]
lgtm
/assign @wongma7

File: pkg/driver/node.go [hunk 1]
```diff
@@ -516,8 +516,7 @@ func (d *nodeService) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoReque
 	zone := d.metadata.GetAvailabilityZone()
 
 	segments := map[string]string{
-		TopologyKey:          zone,
-		WellKnownTopologyKey: zone,
+		TopologyKey: zone,
```
Comment (wongma7) [inline pkg/driver/node.go:519]: this part I don't understand, if the volume gets migrated and its TopologyKey gets translated to WellKnownTopologyKey, how will it get scheduled to any node if all nodes only have TopologyKey? the tests passed so clearly it got scheduled

Comment (jsafrane) [reply pkg/driver/node.go:519]: IMO WellKnownTopologyKey labels are still added to nodes by kubelet. And in future, when there is no cloud provider in kubelet, I don't know who adds them... external cloud provider?

Comment (jsafrane) [reply pkg/driver/node.go:519]: Scratch the previous comment. Scheduler / its PV topology code translates in-tree volumes to CSI:

https://github.com/kubernetes/kubernetes/blob/2bbc0e3c54fdf11b0006efe7f65f6a416ea15d32/pkg/controller/volume/scheduling/scheduler_binder.go#L816

**Comment by jsafrane** [comment]
@wongma7, how do you deal with decreased unit test coverage? -0.008%, that's like one line?

**Comment by wongma7** [comment]
@jsafrane yeah coveralls is too finicky, I tried increasing threshold and will remove it if that doesn't work, I'm always manually merging to get around it anyway.

**Pull request merged**